lib/manager/helmv3/artifacts.ts

```diff
@@ -10,6 +10,20 @@ import {
 } from '../../util/fs';
 import type { UpdateArtifact, UpdateArtifactsResult } from '../types';
 
+async function helmRepoAdd(alias: string, url: string): Promise<void> {
+  const cmd = `helm repo add ${alias} ${url}`;
```

Comment:
Please use `quote()` from package `shlex` to wrap these if there's the possibility that they contain spaces or special characters